src/Illuminate/Database/Connection.php

```diff
@@ -27,21 +28,28 @@ class Connection implements ConnectionInterface
      *
      * @var PDO
      */
-    protected $pdo;
+    private $pdo;
```

Comment:
Just update the phpdoc to PDO|null or something, and people should get the message